deployment/aerie_db_migration.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
+"""Migrate an AERIE Database"""
+
+import os
+import argparse
+import sys
+import shutil
+import subprocess
+import psycopg
```

Comment:
Since we have this dependency now can we include a [requirements.txt](https://pip.pypa.io/en/stable/cli/pip_freeze/#examples) with this script to make the dependencies easier to download? We can note in the docs how to install the dependencies with Pip using requirements.txt.

Reply:
`requirements.txt` has been added